### PR TITLE
WebGPURenderer: PostProcessing + GaussianBlurNode + QuadMesh

### PR DESCRIPTION
**Pull request by sunag**
**Description**

### Simple Depth of Field

**Example**

[Simple Depth of Field Example](https://raw.githack.com/sunag/three.js/dev-gaussianblur/examples/webgpu_skinning_instancing.html)

```js
const scenePass = pass( scene, camera );
const scenePassColor = scenePass.getTextureNode();
const scenePassDepth = scenePass.getDepthNode().remapClamp( .1, .3 );

const scenePassColorBlurred = scenePassColor.gaussianBlur();
scenePassColorBlurred.directionNode = scenePassDepth;

const postProcessing = new PostProcessing( renderer );
postProcessing.outputNode = scenePassColorBlurred;

... loop ...

postProcessing.render();
```

The scene consists of a `RenderPass` that will store the `Color` and `Depth` in that can be used as Nodes for the composition.
The use of `remapClamp()` is to adjust the `DepthNode` range as the strength of blur used in `scenePassColorBlurred.directionNode`.

We could easily connect any other node, or other `RenderPass`, for example to the grayscale effect, we don't need to add another `Pass`, we can just add the node in the same flow, try for example: 

```js
postProcessing.outputNode = scenePassColorBlurred.saturation( 0 );

// or

postProcessing.outputNode = pass( scene, camera ).saturation( 0 );
```

I adapted `GaussianBlur` from `UnrealBloomPass` to create a `GaussianBlurNode`, so we can use it for various purposes.

### Backends

- [x] WebGPU
- [ ] WebGL

### CODE REVIEW

**Comment by Mugen87** [comment]
This is really awesome and shows the potential of node based material system! This will make FX so much more flexible!

**Comment by sunag** [comment]
[Potal using PassNode Example](https://raw.githack.com/sunag/three.js/dev-gaussianblur/examples/webgpu_portal.html)

```js
const material = new MeshBasicNodeMaterial();

// portal
material.colorNode = pass( scenePortal, camera ).context( { getUV: () => viewportTopLeft } );

// opacity effect
material.opacityNode = uv().distance( .5 ).remapClamp( .3, .5 ).oneMinus();
```

**Comment by sunag** [comment]
[Update to QuadMesh](https://github.com/mrdoob/three.js/pull/27369/commits/9ce0b53bf2aae4b4144329c13f6905f3eacb135d)

**Comment by sunag** [comment]
Sorry for the problems with flipY this last month, backends + framebuffers are sometimes confusing in this sense, but with `QuadMesh` the problem was clear, thanks @mugen87 for adding flipY to `webgpu_sandbox`.

File: examples/jsm/objects/QuadMesh.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+		this.setAttribute( 'position', new Float32BufferAttribute( [ - 1, 3, 0, - 1, - 1, 0, 3, - 1, 0 ], 3 ) );
+		this.setAttribute( 'uv', new Float32BufferAttribute( uv, 2 ) );
+
+	}
+
+}
+
+const _geometry = new QuadGeometry();
+
+class QuadMesh {
```
Comment (LeviPesin) [inline examples/jsm/objects/QuadMesh.js:26]: Don't we already have `FullscreenQuad` class? /ping @sunag

Comment (sunag) [reply examples/jsm/objects/QuadMesh.js:26]: It was inspired by this class, but has a few modifications simplifying it to `WebGPURenderer`.

**Pull request merged**